### DISCUSSION TITLE
It looks like I found this mod for version 1.19.4.

### DISCUSSION THREAD

**Discussion by foxiergo** [stashymane/extra-sounds#104] [Show and tell]
### The mod is in this mini-modpack (there are only 4 mods): https://www.curseforge.com/minecraft/modpacks/onlyclientside/files?gameVersionTypeId=4&version=1.19.4

_If you're too lazy, here's a zip with the mod directly._ **Mod updated (or ported) not me!**
[Extra-Sounds-2.3.1+1.19.4~lonefelidae16-build.3.zip](https://github.com/stashymane/extra-sounds/files/11654454/Extra-Sounds-2.3.1%2B1.19.4.lonefelidae16-build.3.zip)

**Comment by foxiergo** [comment]
This mod is also on 1.20.1

**Comment by baduy266** [comment]
I created github account just to say thank you, life saver!

**Comment by pycuda** [comment]
tysm
